packages/vuetify/src/composables/theme.ts

```diff
@@ -0,0 +1,231 @@
+// Utilities
+import { capitalize, computed, inject, provide, ref, watch } from 'vue'
+import { useVuetify } from '@/framework'
+import { colorToInt, colorToRGB, contrastRatio } from '../util'
+
+// Types
+import type { InjectionKey, Ref } from 'vue'
+
+interface BaseColors {
+  background: string
+  surface: string
+  primary: string
+  primaryVariant: string
+  secondary: string
+  secondaryVariant: string
```

Comment:
Following up previous comment, what would be the purpose and difference between secondary and secondaryVariant?  Is it for grading (e.g. like TailwindCSS would)? Or an abstract term with no relationship. What if there would be more than one variant (dark, sephia, loght,...)

Reply:
Its material design color system 
https://material.io/design/color/the-color-system.html#color-theme-creation

Reply:
Oh, right.
So in that naming scheme, it might already have a naming convention w.r.t. telling which color name contrasts enough for text against another.

Reply:
They seem to allow for multiple variants: 
![image](https://user-images.githubusercontent.com/16421948/97985162-f9e69f80-1e2b-11eb-931a-e4b33ce6a61a.png)